usr/lib/linuxmuster-webui/plugins/lmn_linbo_sync/api.py

```diff
@@ -183,19 +184,51 @@ def test_online(host):
 
     :param host: Hostname
     :type host: string
-    :return: OS type
+    :return: OS type (Off, Linbo, OS Linux, OS Windows, OS Unknown)
     :rtype: string
     """
-
-    command=['nmap', '-p', '22', '-O', host]
+    command=["nmap", "-p", "2222,22,135", host, "-oX", "-"]
     r = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=False).stdout.read()
-    if b'Too many fingerprints' in r:
-        return "Linbo"
-    elif b'Linux' in r:
+    xmlRoot = ET.fromstring(r)
+    openPorts = []
+
+    try:
+        hostElement = xmlRoot.findall("host")[0]
+        portsElement = hostElement.findall("ports")[0]
```

Comment:
No need to use `findall` + 0 here, find already do that.

Reply:
Done